_posts/2019-09-02-about-reorgs.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title:  "Reorganizations, failed PoW chains, and L2 mounts"
+date:   2019-09-02 11:11:54 +0200
+categories: sharding internal architecture blockchain distributed mounting
+---
+
+### Intro
+
+Reorganization events is an crucial topic for blockchain architectures and consensus systems.  Coinweb using L2 chains that are inside L1 chains and work across multiple chains, this is especially important.
+
+Coinweb runs on top of proof-of-work consensus systems (PoW), as well as proof-of-stake (PoS) consensus.  The latter usually use consensus algorithms in the BFT family, dBFT, pBFT or variations.  The latter consensus algorithms have _finality_  while PoW consensus has _eventual consistency_.
+
+This article is about PoW L1 chains only.
```

Comment:
We should notice that, as some l1-blockchain uses checkpoints, then they are not purely PoW, but something in between PoW and PoS (from a CAP point of view).